package.json

```diff
@@ -5,7 +5,7 @@
   "main": "bot.js",
   "scripts": {
     "start": "node bot.js",
-    "test": "mocha 'test/**/*.js' "
+    "test": "mocha lib/**/*.test.js"
```

Comment:
I like the idea of tests near the code. Let's keep an eye on how that scales from the perspective of _can you scan the codebase and understand the shape_.